### PR TITLE
Draft or UF2 UNO blog post

### PR DESCRIPTION
**Pull request by mmoskal**
Comments?

### CODE REVIEW

File: docs/blog.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Microsoft MakeCode Blog
 
+## [Bootloader with a fork](/blog/uf2-for-arduino-uno)
+
+October 17th, 2017 by [mmoskal](https://github.com/mmoskal)
+
+UF2 bootloader for Arduino UNO allows for driver-less flashing of an antique board.
```
Comment (pelikhan) [inline docs/blog.md]: Antique... Vintage?

File: docs/blog/uf2-for-arduino-uno.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+it up to a particular Linux embedded board
+and now I was able to scale it down to the AVR level, namely the world-famous
+Arduino UNO. 
+
+The UNO has two AVR chips -- the main one is ATmega328p (the big thing)
+and ATmega16u2 dedicated to handling USB (it was a smaller ATmega8u2 in
+previous iterations of UNO, but is the version with 16k of flash in the most recent
+and common R3).
+
+The main chip sports a 512 byte bootloader (called [optiboot](https://github.com/Optiboot/optiboot))
```
Comment (pelikhan) [inline docs/blog/uf2-for-arduino-uno.md]: supports?

File: docs/blog.md (hunk 1 above)
Comment (mmoskal) [reply docs/blog.md]: Love it!

File: docs/blog/uf2-for-arduino-uno.md (hunk 2 above)
Comment (mmoskal) [reply docs/blog/uf2-for-arduino-uno.md]: sport something-to have or wear something in a proud way

**Pull request merged**